lib/Makefile

```diff
@@ -43,7 +43,7 @@ LIBVER := $(shell echo $(LIBVER_SCRIPT))
 
 DESTDIR?=
 PREFIX ?= /usr/local
-CPPFLAGS= -DXXH_NAMESPACE=LZ4_
+CPPFLAGS= -DXXH_NAMESPACE=LZ4_ -DLZ4_DLL_EXPORT=1
```

Comment:
So it's enabled by default in `Makefile` ?
I'm a bit surprised, as I was expecting this feature to be Visual only.